plugins/catalog-import/src/api/CatalogImportApi.ts

```diff
@@ -16,6 +16,7 @@
 
 import { createApiRef } from '@backstage/core';
 import { PartialEntity } from '../util/types';
+import { GitHubIntegrationConfig } from '@backstage/integration';
```

Comment:
Just had a discussion the other day that this should be a private package that isn't used directly 😅 . The intention is really to provide an API from the app that can be used for SCM access rather than depending on this package directly. There are some dependencies like token generation features and default scm providers that only the app/backend can decide, so we want this logic injected instead.

Reply:
Wouldn't that be the same for the github.com support here only we chose to hardcode it instead of using the configuration. I'd argue that the shortcut was taken before this PR. :)